app/assets/javascripts/controllers/container_deployment/deploy_provider/deploy-provider-review-controller.js

```diff
@@ -0,0 +1,89 @@
+miqHttpInject(angular.module('miq.containers.providersModule')).controller('containers.deployProviderReviewController',
+  ['$rootScope', '$scope',
+  function($rootScope, $scope) {
+    'use strict';
+
+    // Find the data!
+    var next = $scope;
+    while (angular.isUndefined($scope.data)) {
+      next = next.$parent;
+      if (angular.isUndefined(next)) {
+        $scope.data = {};
+      } else {
+        $scope.data = next.wizardData;
+      }
+    }
+
+    $scope.getProviderType = function() {
+      if ($scope.data.providerType == 'openshiftOrigin') {
+        return "OpenShift Origin";
+      } else if ($scope.data.providerType == 'openshiftEnterprise') {
+        return "OpenShift Enterprise";
+      }
+    };
+
+    $scope.getProviderDescription = function() {
+      if ($scope.data.provisionOn == 'existingVms') {
+        var provider = $scope.data.providers.find(function(provider) {
+          return provider.id == $scope.data.existingProviderId;
+        });
+        return 'Use existing VMs from an existing provider: ' + (provider ? provider.name : "None");
+      }
+      else if ($scope.data.provisionOn == 'newVms') {
+        var provider = $scope.data.providers.find(function(provider) {
+          return provider.id == $scope.data.newVmProviderId;
+        });
+        return 'Use existing VMs from an existing provider: ' + (provider ? provider.name : "None");
+      }
+      else if ($scope.data.provisionOn == 'noProvider') {
+        return 'Specify a list of machines to deploy on (No existing provider';
+      }
+    };
+
+    $scope.getMasterCreationTemplate = function () {
+      var selectedTemplate = $scope.data.nodeCreationTemplates.find(function(nextTemplate) {
+        return nextTemplate.id == $scope.data.masterCreationTemplateId;
+      });
+      return selectedTemplate ? selectedTemplate.name : "None";
+    };
+
+    $scope.getNodeCreationTemplate = function () {
+      var selectedTemplate = $scope.data.nodeCreationTemplates.find(function(nextTemplate) {
+        return nextTemplate.id === $scope.data.masterCreationTemplateId;
+      });
+      return selectedTemplate ? selectedTemplate.name : "None";
+    };
+
+    $scope.getAuthenticationType = function () {
```

Comment:
I mean all the returned strings in getAuthenticationType, since they are shown to the user.. I agree that translating `Google` may make less sense than translating `Request Header`, but that should be up to the translators, not us..